[GENFI] Update genfi patch (#912)
aramis-lab/clinica: docs/Converters/GENFItoBIDS.md

- Fieldmaps
- rsfMRI

## Downloading GENFI

> To download GENFI in a way that you can convert it, you only need to make sure of two thing: 

Reviewer: ```suggestion
To download GENFI in a way that you can convert it, you need to make sure of two things: 
```

> - To only select the "simplify downloaded archive structure" in download data options.

Reviewer: ```suggestion
- Only select the "simplify downloaded archive structure" in download data options.
```

> - And in Select Image Data section Scans Format, select only "DICOM"

Reviewer: I'm not sure my suggestion is correct, please verify where the `"` should be placed.

```suggestion
- Select only "DICOM" in "Select Image Data" section "Scans Format".
```
